cmd/swarm-rafttool/common.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"os"
+
+	"github.com/coreos/etcd/pkg/fileutil"
+	"github.com/coreos/etcd/wal/walpb"
+	"github.com/docker/swarmkit/ca"
+	"github.com/docker/swarmkit/manager"
+	"github.com/docker/swarmkit/manager/encryption"
+	"github.com/docker/swarmkit/manager/state/raft/storage"
+)
+
+func certPaths(swarmdir string) *ca.SecurityConfigPaths {
+	return ca.NewConfigPaths(filepath.Join(swarmdir, "certificates"))
+}
+
+func getDEKData(krw *ca.KeyReadWriter) (manager.RaftDEKData, error) {
+	h, _ := krw.GetCurrentState()
+	dekData, ok := h.(manager.RaftDEKData)
+	if !ok {
+		return manager.RaftDEKData{}, fmt.Errorf("cannot read raft dek headers in TLS key ")
```

Comment:
Extra space at the end of the message, and these returns can use `errors.New` since there is no format string.